Source/Classes/PINRemoteImageDownloadQueue.m

```diff
@@ -68,19 +68,21 @@ - (NSURLSessionDataTask *)addDownloadWithSessionManager:(PINURLSessionManager *)
                                                priority:(PINRemoteImageManagerPriority)priority
                                       completionHandler:(PINRemoteImageDownloadCompletion)completionHandler
 {
-    NSURLSessionDataTask *dataTask = [sessionManager dataTaskWithRequest:request completionHandler:^(NSURLSessionTask *task, NSError *error) {
-        completionHandler(task.response, error);
-        [self lock];
-            [self->_runningTasks removeObject:task];
-        [self unlock];
-        
-        [self scheduleDownloadsIfNeeded];
-    }];
-    
+    NSURLSessionDataTask *dataTask = [sessionManager dataTaskWithRequest:request
+                                                                priority:priority
+                                                       completionHandler:^(NSURLSessionTask *task, NSError *error) {
+                                                           completionHandler(task.response, error);
+                                                           [self lock];
```

Comment:
Do we need the indentation change here?

Reply:
The new priority param made this line too long so I break it into multiple lines hence the new indentation.